config/logit/README.md

```diff
@@ -0,0 +1,64 @@
+# What
+
+This directory contains the logstash filters to be used in logit.io to parse
+the logs from our Cloud Foundry platform.
+
+The scripts are based on
+ - Upstream [logsearch filters](https://github.com/logsearch/logsearch-boshrelease/tree/develop/src/logsearch-config/src/logstash-filters)
+ - [logsearch-for-cloudfoundry filters](https://github.com/alphagov/paas-logsearch-for-cloudfoundry)
+ - Custom filters added adhoc by us.
+
+These filters can be manually updated in the logit interface. To generate the filters or update them, follow the instructions in this file.
+
+## How to generate the Logit Logstash filters
+
+Run `scripts/generate_logit_filters.sh` from the paas-cf directory.
+
+### 10\_base.conf
+
+This contains the base filters.
+
+Concatenate the following files:
+
+* [filters_pre.conf.erb](https://github.com/cloudfoundry-community/logsearch-boshrelease/blob/v205.0.1/jobs/parser/templates/config/filters_pre.conf.erb)
+* [redact_passwords.conf](https://github.com/cloudfoundry-community/logsearch-boshrelease/blob/v205.0.1/src/logsearch-config/src/logstash-filters/snippets/redact_passwords.conf)
+* [syslog_standard.conf](https://github.com/cloudfoundry-community/logsearch-boshrelease/blob/v205.0.1/src/logsearch-config/src/logstash-filters/snippets/syslog_standard.conf)
+
+Apply the following changes:
+ - remove the ERB template blocks
+ - replace the first alter {} block with this:
+   ```
+   mutate {
+     add_field => [ "type", "syslog" ]
+   }
+   ```
+  - remove all ruby {} blocks
+
+### 20\_logsearch\_for\_cloudfoudry.conf
```

Comment:
When I follow these instructions I get a different result to that stored in `config/logit/20_logsearch_for_cloudfoundry.conf`. I am using the latest master of paas-logsearch-for-cloudfoundry.

Diff: https://gist.github.com/henrytk/00866a02dccb06db6b04565f00e6c75b